FusionIIIT/applications/research_procedures/views.py

```diff
@@ -714,11 +709,7 @@ def add_staff_request(request,id):
         projectid = int(id)
         receiver_designation = obj.get('receiver')
 
-        if receiver_designation == 'Assistant':
-            receiver_designation = 'Assistant Professor'
         
-        if receiver_designation == 'CSE':
-            receiver_designation = 'CSE HOD'
         
         receiver_designation= get_designation_instance(receiver_designation)
```

Comment:
Ig after removing the above lines of code, this will give an `null` instance and the code functionality will break.